src/browser.d.ts

```diff
@@ -0,0 +1,25 @@
+type COSESign1 = {
+  signature: string;
+  key: string;
+}
+
+type Signer = (msg: string) => PromiseLike<COSESign1>;
+
+export function sign(
```

Comment:
Can we stick with arrow function types? Also we should make first letter of the function uppercase ⏫